src/tests/class_tests/openms/source/AbsoluteQuantitationMethodFile_test.cpp

```diff
@@ -261,6 +261,34 @@ START_SECTION((void load(const String & filename, std::vector<AbsoluteQuantitati
 
 END_SECTION
 
+START_SECTION(void store(const String & filename, const std::vector<AbsoluteQuantitationMethod> & aqm_list) const)
+  AbsoluteQuantitationMethodFile aqmf;
+  vector<AbsoluteQuantitationMethod> aqm_list1, aqm_list2;
+  aqmf.load(OPENMS_GET_TEST_DATA_PATH("AbsoluteQuantitationMethodFile_1.csv"), aqm_list1);
+  aqmf.store(OPENMS_GET_TEST_DATA_PATH("AbsoluteQuantitationMethodFile_2.csv"), aqm_list1);
+  aqmf.load(OPENMS_GET_TEST_DATA_PATH("AbsoluteQuantitationMethodFile_1.csv"), aqm_list2);
```

Comment:
shouldnt this load ` "AbsoluteQuantitationMethodFile_2.csv"` in order to really test the store method?

Reply:
Yeah, of course it should! Thanks for catching this. During the tests I kept looking at the files outputs and they were identical except for the end of lines (no '\r' in the output files). Didn't notice this bug.